fixture/src/contacts/data/contacts.ts

```diff
@@ -411,8 +411,8 @@ const contacts = contactsData.reduce((contactsMap, contact) => {
   return contactsMap;
 }, new Map<string, Contact[]>());
 contacts.forEach(
-  (contacts: Contact[], key: string, map: Map<string, Contact[]>) => {
-    const sortedContacts = contacts.sort((aContact, bContact) =>
+  (_contacts: Contact[], key: string, map: Map<string, Contact[]>) => {
+    const sortedContacts = _contacts.sort((aContact, bContact) =>
```

Comment:
Not sure if this change was necessary since we don't want to access the outer `contacts` but I'm not against keeping it this way as well.